fastcrypto/src/groups/bls12381.rs

```diff
@@ -533,6 +534,19 @@ impl ScalarType for Scalar {
     }
 }
 
+impl HashToGroupElement for Scalar {
+    fn hash_to_group_element(bytes: &[u8]) -> Self {
+        let digest = Sha256::digest(bytes); // TODO: should this be 512?
```

Comment:
Interestingly, for Ristretto we used 512 and here 256. Let's make them aligned

Reply:
I think it should be 512 in both, but I used here 256 to be consistent with https://github.com/MystenLabs/fastcrypto/blob/main/fastcrypto/src/bls12381/min_sig/mskr.rs#L25-L26

@jonas-lj what was the reason we used 256 in mskr?

Reply:
It's a while back, but I think there was a discussion if it was sufficient with half length scalars for mskr, in which case SHA-256 truncated to 128 bits should be fine for that specific case.